cluster/store/store.go

```diff
@@ -638,7 +643,24 @@ func (st *Store) Apply(l *raft.Log) interface{} {
 		panic("error proto un-marshalling log data")
 	}
 
+	schemaOnly := l.Index <= st.initialLastAppliedIndex
 	defer func() {
+		if !st.dbAndRaftAligned.Load() && l.Index == st.initialLastAppliedIndex {
```

Comment:
There are probably multiple ways to detect we have caught with the at-start state. Happy to talk about a better approach.

I'm thinking of
```
Let's avoid the new atomic and instead detect only l.Index == st.initialLastAppliedIndex. With good comments it should be clear enough
```

Reply:
i agree, maybe let's avoid having another flag